src/main/java/spireMapOverhaul/zones/candyland/CandyLand.java

```diff
@@ -28,12 +29,12 @@
 import spireMapOverhaul.zones.candyland.consumables.rare.GoldCandy;
 import spireMapOverhaul.zones.candyland.consumables.uncommon.*;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
 
-public class CandyLand extends AbstractZone implements RewardModifyingZone, CampfireModifyingZone, ShopModifyingZone {
+public class CandyLand extends AbstractZone implements RewardModifyingZone, RenderableZone, CampfireModifyingZone, ShopModifyingZone {
     public static final String ID = "CandyLand";
+    private Texture CandylandBackground = new Texture("anniv6Resources/images/backgrounds/candyland/CandyLandBackground.png");
```

Comment:
This is how you create a memory leak, every time this zone is created it will create this texture and never dispose of them so they'll be in memory.

Please use `TexLoader.getTexture(SpireAnniversary6Mod.makeImagePath("backgrounds/candyland/CandyLandBackground.png"))` or something close to it, that way the same texture will be reused.